dm/master/scheduler/scheduler.go

```diff
@@ -1548,11 +1556,14 @@ func (s *Scheduler) tryBoundForWorker(w *Worker) (bounded bool, err error) {
 
 	// randomly pick one from unbounds
 	if source == "" {
-		for source = range s.unbounds {
-			s.logger.Info("found unbound source when worker bound",
-				zap.String("worker", w.BaseInfo().Name),
-				zap.String("source", source))
-			break // got a source.
+		for unboundedSource := range s.unbounds {
```

Comment:
after `recoverWorkersBounds`, not existing sources are not put in `s.unbounds`, so I think this logic could be removed